### ISSUE TITLE
internal_logs is documented as being available in vector 0.11

### ISSUE THREAD

**Issue by szibis** [vectordotdev/vector#5977]
I see that internal_logs are merged in Vector also documentation says https://vector.dev/docs/reference/sources/internal_logs/ that it should be available from 0.11 but docker alpine 0.11.1 not supporting that command.
I checked the latest nightly and the source is available there but not in 0.11.1

```
Jan 11 11:33:23.603  INFO vector::app: Loading configs. path=[("/etc/vector/vector.toml", Some(TOML))]
Jan 11 11:33:23.607 ERROR vector::cli: Configuration error. error=unknown variant `internal_logs`, expected one of `apache_metrics`, `aws_ecs_metrics`, `aws_kinesis_firehose`, `aws_s3`, `docker`, `docker_logs`, `file`, `generator`, `host_metrics`, `http`, `internal_metrics`, `journald`, `kafka`, `kubernetes_logs`, `logplex`, `mongodb_metrics`, `nginx_metrics`, `prometheus`, `prometheus_remote_write`, `prometheus_scrape`, `socket`, `splunk_hec`, `statsd`, `stdin`, `syslog`, `vector` for key `sources.internal_logs.type` at line 32 column 10
```

**Comment by jszwedko** [comment]
I believe this change wasn't actually released in 0.11.1, it should be released in 0.12.0 though.

@binarylogic would it be possible to regenerate the docs against 0.11.1? I think there are a few items on the website that haven't technically been released yet. For example, TCP keepalive which a discord user noted: https://discord.com/channels/742820443487993987/746070591097798688/795897415848427600

**Comment by juchiast** [comment]
btw, `docker_host` option is not in 0.11.1 too.

**Comment by skupjoe** [comment]
I am trying to perform a `filter` transform on `internal_logs` using the latest vector aggregator nightly in the same way that it was working before.

I am now getting an error `ERROR vector::cli: Configuration error. error=missing field 'type' for key 'transforms.vector_errors' at line 171 column 1`

Here is the relevant config for that transform from my `values.yaml`:

```
  vector_errors:
    type: "filter"
    inputs: ["internal_logs"]
    rawConfig: |
      condition."metadata.level.not_contains" = [
        "INFO",
        "DEBUG",
        "TRACE"
      ]
```

Is it possible that this is related?

**Comment by bruceg** [comment]
> I am now getting an error `ERROR vector::cli: Configuration error. error=missing field 'type' for key 'transforms.vector_errors' at line 171 column 1`

This is due to #5978. All conditions now require a `type` field.

**Comment by jamtur01** [comment]
Please re-open if you find a further issue! Thanks!

**Comment by jszwedko** [comment]
Reopening this to represent the fact that `internal_logs` source is not available in vector 0.11, but is documented as available: https://vector.dev/docs/reference/sources/internal_logs/#requirements
